obal/data/roles/mock/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Build with mock
+  shell: "mock --recurse --chain -r {{ mock_config }} --localrepo {{ inventory_dir }}/mock_builds SRPMs/*"
```

Comment:
shall `mock_builds` be configurable?

Reply:
I suppose it could be if there is a use case for making it configurable.

Reply:
mostly me not liking static stuff :)
maybe pointing this at `~/public_html/myrepo` but then we'd have to drop `{{ inventory_dir }}` too

Reply:
I will make it configurable given that is reasonable to use the public_html location. But I want to keep the default within the project itself to prevent overlap of mock builds across multiple repositories on the same machine (yes, I do this today).